class.jetpack-debugger.php

```diff
@@ -230,6 +230,10 @@ public static function jetpack_debug_display_handler() {
 
 		$tests = array();
 
+		$tests['XML']['result'] = ( function_exists( 'xml_parser_create' ) ) ? 'PASS' : false;
+		/* translators: Link to Jetpack Hosting support page. */
+		$tests['XML']['fail_message'] = __( sprintf( 'Jetpack can not load necessary XML manipulation libraries. Please ask your hosting provider to refer to our <a href="%s">server requirements</a>.', 'https://jetpack.com/support/server-requirements/' ), 'jetpack' );
```

Comment:
I would recommend doing this instead:

```suggestion
		$tests['XML']['fail_message'] = sprintf( __( 'Jetpack can not load necessary XML manipulation libraries. Please ask your hosting provider to refer to our <a href="%s">server requirements</a>.', 'jetpack' ), 'https://jetpack.com/support/server-requirements/' );
```